app/src/services/dataService.js

```diff
@@ -269,5 +269,32 @@ module.exports = {
       }
     }
     return await this.getIPCPlan(ipcPlanId);
+  },
+
+  async update(business, contacts, ipcPlan, location, updatedBy) {
+    try {
+      await db.sequelize.transaction(async t => {
+        // currently, we are only allowed to update location data
+        // get the current data...
+        const currentLocation = await db.Location.findByPk(location.locationId);
+        // update the allowed fields...
+        currentLocation.startDate = location.startDate;
```

Comment:
What if I only update the endDate, will this overwrite the startDate with blank? Or will we have an update "form" that contains all the fields and guarantee that it's atomic at the all-fields level?

Reply:
This PR is only a PoC until we know what the front end will be for updates...

Reply:
👍